tests/karma/BookNavigator/volumes/volumes-provider.test.js

```diff
@@ -10,18 +10,21 @@ const brOptions = {
         "/details/SubBookTest": {
           "url_path": "/details/SubBookTest",
           "file_subprefix": "book1/GPORFP",
+          "orig_sort": 1,
```

Comment:
Is this name set in stone? `orig` is kind of odd, since we don't know when "orig" was set, or what it means. Can this be `"index"`?

Reply:
we can change it to index in PB side i think

Reply:
keeping `orig_sort` for now

Reply:
@cdrini it looks like an index, but if we change algo from under, it will no longer be an index.  I think it's safe to keep as `orig_sort` for now to decouple from actual `index